py3status/modules/scratchpad_async.py

```diff
@@ -37,6 +35,17 @@ class Meta:
                     'msg': '{} should not be used in format use `{counter}`',
                 },
             ],
+            'substitute_by_value': [
+                {
+                    'param': 'always_show',
+                    'value': True,
+                    'substitute': {
+                        'param': 'format',
+                        'value': '{counter} ⌫',
+                    },
+                    'msg': 'obsolete parameter. update format instead',
+                },
```

Comment:
Damn. You saw that. Grr. Just grab the whole thing and `[`,`]` it?
Modules shouldn't try to be clever or not in that way - it leads to madness. 👼